bodhi/server/buildsys.py

```diff
@@ -315,12 +315,16 @@ def get_krb_conf(config):
 def get_session():
     """ Get a new buildsystem instance """
     global _buildsystem
-    if not _buildsystem:
-        log.warning('No buildsystem configured; assuming testing')
-        return DevBuildsys()
+    assert _buildsystem is not None, 'Buildsys needs to be setup'
```

Comment:
```assert``` statements get stripped out in .pyo files - it's important to raise an Exception since we want the production code to behave this way.